components/annot-age-values.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <b-card no-body>
+      <b-card-header>I am a header</b-card-header>
+      <b-card-body style="position:relative; height:300px; overflow-y:scroll;">
```

Comment:
Move to style tag below for this class or create an id for this element to reference in CSS